src/index.js

```diff
@@ -33,15 +34,16 @@ const useReduxForm = memoize(({ layout, config }) => {
         }
         if (render) {
           return (
-            <div key={itemKey}>
+            <>
               {render({
+                key: itemKey,
                 disabled: formProps.disabled,
                 readonly: formProps.readonly,
                 size: formProps.size,
                 ...itemProps,
                 formProps,
               })}
-            </div>
+            </>
```

Comment:
Why are we changing this?

Reply:
I don't think this should be a div because it affects the css

https://stackoverflow.com/questions/47761894/why-are-fragments-in-react-16-better-than-container-divs

point 2 specifically

Reply:
What css affects are we talking about? Give me examples.

If you "send" the `key` to `render`, you are passing the responsibility of specifying the `key` to consumer, which they will either forget, or find it confusing to do so, b/c they don't have the context of rendering multiple elements only visible in `redux-form-sui`. Of course, I had no choice but to do this for `render` in `renderSection` and few others, and I left a very comprehensive comment on that too.

Reply:
ohhh i see

I've not been that conscientious of my div vs fragment usage so this was a new thing for me to keep in mind.

One example I came across that led me to this was when I tried to make a flexbox inside the form header. I needed the  width to be 100% but because this div was in between my container div (which holds the form) and my flexbox div, I could not get the width to 100%. 

Maybe an alternate solution could be to allow styling on this div? I also considered that but because we already have `style, css` both of which are applied to this divs parent so I didn't want to have even more props that make it complicated.

An alternate alternate solution could be maybe sending in a `& div' styling to the css to target this div?

What do you think?

Reply:
I can't visualize your example, maybe we can get on a call to demo